### PR TITLE
ref(getting-started-docs): Add configure source maps step

### PR DESCRIPTION
**Pull request by priscilawebdev**
Add a new step titled "Configure Source Maps" to the Getting Started layout. 

This PR:
- Adds this step to the React Documentation and will soon be incorporated into other JavaScript docs.
- Remove "source maps" as next step

**Preview:**

image

### CODE REVIEW

File: static/app/gettingStartedDocs/javascript/react.tsx [hunk 1]
```diff
@@ -69,6 +71,27 @@ export const steps = ({
       },
     ],
   },
+  {
+    language: 'html',
```
Comment (priscilawebdev) [inline static/app/gettingStartedDocs/javascript/react.tsx:75]: when this is `javascript` the format looks a bit strange and there are no colors... we'll have to see support for npx

Comment (matejminar) [reply static/app/gettingStartedDocs/javascript/react.tsx:75]: how about language `bash`?

File: static/app/components/onboarding/gettingStartedDoc/step.tsx [hunk 2]
```diff
@@ -6,17 +6,22 @@ import {t} from 'sentry/locale';
...
+   */
+  CONFIGURE_SOURCE_MAPS = 'configure_source_maps',
   VERIFY = 'verify',
 }
 
 export const StepTitle = {
   [StepType.INSTALL]: t('Install'),
-  [StepType.CONFIGURE]: t('Configure'),
+  [StepType.CONFIGURE_SDK]: t('Configure SDK'),
+  [StepType.CONFIGURE_SOURCE_MAPS]: t('Configure Source Maps (Automatic Setup)'),
```
Comment (matejminar) [inline static/app/components/onboarding/gettingStartedDoc/step.tsx:20]:
```suggestion
  [StepType.CONFIGURE_SOURCE_MAPS]: t('Configure Source Maps'),
```

File: static/app/gettingStartedDocs/javascript/react.tsx [hunk 3]
```diff
@@ -69,6 +71,27 @@ export const steps = ({
...
+    type: StepType.CONFIGURE_SOURCE_MAPS,
+    configurations: [
+      {
+        description: t(
+          'The easiest way to configure upload Source Maps with Webpack is by using the Sentry Wizard.'
```
Comment (matejminar) [inline static/app/gettingStartedDocs/javascript/react.tsx:80]: We should not mention webpack here as we don't know what they are using (wizard will let them choose this).

File: static/app/gettingStartedDocs/javascript/react.tsx [hunk 4]
```diff
@@ -69,6 +71,27 @@ export const steps = ({
...
+        ),
+        code: `npx @sentry/wizard@latest -i sourcemaps`,
+        additionalInfo: (
+          <Alert type="info" showIcon noBottomSpacing>
+            {tct('Prefer to set up Source Maps manually? [docsLink:Read the docs].', {
+              docsLink: (
+                <ExternalLink href="https://docs.sentry.io/platforms/javascript/sourcemaps/uploading/webpack/#manual-setup" />
```
Comment (matejminar) [inline static/app/gettingStartedDocs/javascript/react.tsx:87]: We should link to the generic one, not the webpack one.
Probably just here:
https://docs.sentry.io/platforms/javascript/sourcemaps/uploading/

File: static/app/gettingStartedDocs/javascript/react.tsx (hunk 1 above)
Comment (Lms24) [reply static/app/gettingStartedDocs/javascript/react.tsx:75]: `language` is for syntax highlighting of the `npx` command, right? Can we set something like `bash` or `sh` here? Maybe this is more accurate than html

**Review by Lms24 — APPROVED** [review]
Nice, LGTM!

**Comment by priscilawebdev** [comment]
updated :) 

image

File: static/app/gettingStartedDocs/javascript/react.tsx [hunk 5]
```diff
@@ -69,6 +71,27 @@ export const steps = ({
...
+      {
+        description: t(
+          'Use Sentry Wizard to configure Source Maps upload, enabling readable stack traces in Sentry errors.'
+        ),
+        code: `npx @sentry/wizard@latest -i sourcemaps`,
+        additionalInfo: (
+          <Alert type="info" showIcon noBottomSpacing>
+            {tct('Prefer to set up Source Maps manually? [docsLink:Read the docs].', {
+              docsLink: (
+                <ExternalLink href="https://docs.sentry.io/platforms/javascript/sourcemaps/uploading/" />
```
Comment (matejminar) [inline static/app/gettingStartedDocs/javascript/react.tsx:87]:
```suggestion
                <ExternalLink href="https://docs.sentry.io/platforms/javascript/guides/react/sourcemaps/" />
```

**Review by matejminar — APPROVED** [review]

**Comment by danielkhan** [comment]
> updated :)

The text is a bit clunky.

Maybe:

"Upload Source Maps"

"Automatically upload your source maps to enable readable stack traces for Errors. If you prefer to manually set up source maps, please follow [this guide](link to guide)."

**Comment by Jesse-Box** [comment]
@priscilawebdev for the copy, id suggest the following

Section Title:
```
Upload Source Maps
```
Section Text:
```
Automatically upload your source maps to enable readable stack traces for Errors. If you prefer to manually set up source maps, please follow [this guide](link to guide).
```

**Comment by priscilawebdev** [comment]
feedback applied!

**Pull request merged**